requirements.txt

```diff
@@ -2,7 +2,7 @@ ark-analysis @ git+https://github.com/angelolab/ark-analysis.git@02b4549731c2204
 mibi-bin-tools @ git+https://github.com/angelolab/mibi-bin-tools.git@master
```

Comment:
These should both be changed to the tagged version, rather than master, and for `ark`, we can pip install